spec/models/concerns/current_tracked_by_changeset_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe CurrentTrackedByChangeset do
+  let(:stop) { create(:stop) }
+
+  context '#as_change' do
+    it 'converts to camelCase' do
+      expect(stop.as_change[:onestopId]).to eq(stop.onestop_id)
+    end
+
+    it 'only includes changeable attributes' do
+      expect(stop.as_change[:createdAt]).to be nil
+      expect(stop.as_change[:updatedAt]).to be nil
+    end
+
+    it 'includes virtual attributes' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.